docs/tutorials/python/profiler.md

```diff
@@ -164,28 +150,33 @@ profiler.set_state('run')
 
 run_training_iteration(*next(itr))
 
-# Ask the profiler to stop recording after operations have completed
+# Make sure all operations have completed
 mx.nd.waitall()
+# Ask the profiler to stop recording
 profiler.set_state('stop')
 ```
 
 Between running and stopping the profiler, you can also pause and resume the profiler using `profiler.pause()` and `profiler.resume()` respectively to profile only parts of the code you want to profile.
 
-### Starting profiler automatically using environment variable
+### Starting the profiler automatically using an environment variable
 
 The method described above requires code changes to start and stop the profiler. You can also start the profiler automatically and profile the entire code without any code changes using the `MXNET_PROFILER_AUTOSTART` environment variable.
 
-MXNet will start the profiler automatically if you run your code with the environment variable `MXNET_PROFILER_AUTOSTART` set to `1`. The profiler output is stored into `profile.json` in the current directory.
+`$ MXNET_PROFILER_AUTOSTART=1 python my_script.py`
+
+MXNet will start the profiler automatically if you run your code with the environment variable `MXNET_PROFILER_AUTOSTART` set to `1`. The profiler output is stored in `profile.json` inside the current directory.
 
 Note that the profiler output could be large depending on your code. It might be helpful to profile only sections of your code using the `set_state` API described in the previous section.
 
 ### Increasing granularity of the profiler output
 
 MXNet executes computation graphs in 'bulk mode' which reduces kernel launch gaps in between symbolic operators for faster execution. This could reduce the granularity of the profiler output. If you need profiling result of every operator, please set the environment variables `MXNET_EXEC_BULK_EXEC_INFERENCE` and `MXNET_EXEC_BULK_EXEC_TRAIN` to `0` to disable the bulk execution mode.
 
+When working with Gluon networks, you will get a more granular output if you profile without hybridization.
```

Comment:
When working with networks created using the Gluon API, .... 

Also,what do you mean granular - this may be hard for a non-native speaker to understand what you mean. I would spell it out saying the logging output is going to be updated more frequently than if you use symbolic where the logging output will be outputed less frequently but both will contain the same information.

Reply:
done.